src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -514,8 +488,35 @@ public NamedOnnxValue GetNamedOnnxValue()
     }
 
     /// <summary>
-    /// A class implementing the estimator interface of the <see cref="OnnxTransformer"/>.
+    /// <see cref="IEstimator{TTransformer}"/> for scoring ONNX models in the ML.NET framework.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Known-sized vector of <xref:System.Single> or <xref:System.Double> types |
+    /// | Output column data type | As specified by the ONNX model |
+    /// | Required NuGet in addition to Microsoft.ML | Microsoft.ML.OnnxRuntime (always),  Microsoft.ML.OnnxRuntime.Gpu (only if using GPU device) |
```

Comment:
> Microsoft.ML.OnnxRuntime (always),  Microsoft.ML.OnnxRuntime.Gpu (only if using GPU device)  [](start = 54, length = 93)

this is the correct one:

Microsoft.ML.OnnxTransformer (always), Microsoft.ML.OnnxRuntime.Gpu (only if GPU processing is used)